src/background/messaging/rpc-methods/send-transfer.ts

```diff
@@ -43,8 +56,7 @@ export async function rpcSendTransfer(message: SendTransferRequest, port: chrome
   }
 
   const requestParams: RequestParams = [
-    ['address', message.params.address],
-    ['amount', message.params.amount],
+    ['recipients', JSON.stringify(params.recipients)],
```

Comment:
I think we will have to handle being backward compatible here so we don't break anyone using the old api?

Reply:
I convert old params to new ones above